bundles/org.openhab.binding.plugwiseha/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -88,6 +112,10 @@
 		<label>Flame State</label>
 		<description>Is the boiler's flame active, On or OFF</description>
 		<category>switch</category>
+		<tags>
+			<tag>Status</tag>
+			<tag>Power</tag>
```

Comment:
Not sure. The heater is always powered on, it is just not ignited/burning. Maybe Status+Heating ?!